### DISCUSSION TITLE
post/2022-07-21-grpc-interceptor-call-option/

### DISCUSSION THREAD

**Discussion by giscus** [zcong1993/blog#71] [Announcements]
# post/2022-07-21-grpc-interceptor-call-option/

本文介绍 Grpc client 拦截器 CallOption 扩展.

https://blog.cong.moe/post/2022-07-21-grpc-interceptor-call-option/

**Comment by Shinena1998** [comment]
感觉在option中传值比context要优雅些

**Reply by zcong1993** [reply]
那肯定的
